src/components/LayoutFooter/styles.module.css

```diff
@@ -102,20 +110,151 @@
 
   display: flex;
   align-items: center;
-  flex-wrap: nowrap;
-  line-height: 23px;
-  font-size: 20px;
-  color: #fff;
+  font-size: 18px;
+  line-height: 30px;
+  color: rgba(255, 255, 255, 0.75);
   text-decoration: none;
+  letter-spacing: 0.02em;
+  padding: 4px 0;
+
+  @media (--md-scr) {
+    font-size: 16px;
+    line-height: 28px;
+  }
 
   &:hover {
-    color: #ccc;
+    color: #fff;
   }
 }
 
 .icon {
   flex-shrink: 0;
-  margin-right: 14px;
+  margin-right: 12px;
+  width: 24px;
+  height: 24px;
+
+  @media (--md-scr) {
+    margin-right: 10px;
+  }
+}
+
+.productIcon {
+  flex-shrink: 0;
+  margin-right: 12px;
+  width: 20px;
+  height: 20px;
+
+  @media (--md-scr) {
+    margin-right: 10px;
+    width: 18px;
+    height: 18px;
+  }
+}
+
+.bottomRow {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+
+  @media (--sm-scr) {
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+  }
+}
+
+.companyLabel {
+  font-size: 18px;
+  line-height: 30px;
+  color: rgba(255, 255, 255, 0.75);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.companyDescription {
+  @media (--xs-scr) {
+    font-size: 14px;
+    line-height: 16px;
+    width: 100%;
+  }
+}
+
+.companyName {
+  @mixin active;
+  @mixin hover;
+
+  font-weight: 800;
```

Comment:
While the font-weight says 800, I'm pretty sure it's 600 since that appears to be the largest weight our font has: 
![image](https://user-images.githubusercontent.com/43496356/119194445-b01c7a00-ba48-11eb-9952-5d9d31c479ee.png)